src/intent.html

```diff
@@ -305,7 +305,7 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
   <section>
    <h3 id="intent_using">Using Intent Concepts and Properties</h3>
    <p>When the <code class="attribute">intent</code> attribute corresponding to a specific node
-     contains a concept component, the AT's [=Intent Concept Dictionary=] should be consulted.
+     contains a concept component, the supported [=Intent Concept Dictionary=] should be consulted.
```

Comment:
I think this is better with AT as currently written, otherwise it is not saying what is supporting this. The important point here is that thi sis system (AT) specific support,

Reply:
I thought I heard you say you prefer that word choice during the meeting today when reviewing a change I had made? No matter, I have removed it to avoid distractions.